### PR TITLE
Switched add/remove user_class functions in e_user_model to make use of the unified functions

### PR DESCRIPTION
**Pull request by SimSync**
from the user_class class (user_class::ucAdd(), user_class::ucRemove())
Fixed some typo and removed missing template vars

### CODE REVIEW

**Comment by CaMer0n** [comment]
Thank you! I have created an empty test for you. 
https://github.com/e107inc/e107-test/commit/6bba285eeebebf0e5677ffa2cb9bac846d9390e1

**Comment by CaMer0n** [comment]
@SimSync

**Comment by SimSync** [comment]
@CaMer0n Done 😏

File: e107_handlers/user_model.php [hunk 1]
```diff
@@ -1285,10 +1297,13 @@ public function renderEmail($type, $userInfo)
...
-			$EMAIL_TEMPLATE['quickadduser']['body']		= $QUICKADDUSER_TEMPLATE['email_body'];
-			$EMAIL_TEMPLATE['notify']['body']			= $NOTIFY_TEMPLATE['email_body'];
-			
+
+			// Both template vars are not defined...
+			//$EMAIL_TEMPLATE['quickadduser']['body']		= $QUICKADDUSER_TEMPLATE['email_body'];
+			//$EMAIL_TEMPLATE['notify']['body']			= $NOTIFY_TEMPLATE['email_body'];
+			$EMAIL_TEMPLATE['quickadduser']['body']	= '';
+			$EMAIL_TEMPLATE['notify']['body']			= '';
+
```
Comment (CaMer0n) [inline e107_handlers/user_model.php]: Thank you!. Could you explain the thinking behind setting `$EMAIL_TEMPLATE['quickadduser']['body'] ` etc. to empty values?

Comment (SimSync) [reply e107_handlers/user_model.php]: This was just a quick fix for the missing variables.
I did a search on the whole codebase and i couldn't find a var called `$QUICKADDUSER_TEMPLATE` or `$NOTIFY_TEMPLATE`. So instead making both unknown vars global to be able to use them in the method, i decided to just remove them by commenting out and setting an empty string instead.

Do you know, where to find those vars or what content to use instead for the `$EMAIL_TEMPLATE` var?

Comment (CaMer0n) [reply e107_handlers/user_model.php]: I think it's a BC fix. We should probably check if `$QUICKADDUSER_TEMPLATE['email_body']` is not empty, and assign it to `$EMAIL_TEMPLATE['quickadduser']['body']` if it has a value. Similar with 'notify'

Comment (SimSync) [reply e107_handlers/user_model.php]: I will make that missing vars global and use vartrue() for the assignment.
But first I must sort out my line endings issue... Don't want to screw up another file...

Comment (SimSync) [reply e107_handlers/user_model.php]: @CaMer0n Pushed a new commit with the proposed changes to the 2 vars.

**Pull request merged**